<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clienttag</class>
 <widget class="QWidget" name="clienttag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Client Tags:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tagAddButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="availableTagsList">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>171</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="tagRemoveButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tagCreateButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QListWidget" name="clientTagsList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>181</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>(Available)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
